styleguide/javascript/README.md

```diff
@@ -0,0 +1,140 @@
+# JavaScript Styleguide
+
+## Tooling
+
+#### Avoid jQuery
```

Comment:
Intentionally skipping a header level?

Reply:
Not intentionally but not by mistake either. Think I just followed the levels that were in the other react styleguide. Fixing :)